Doras/src/app/carousel/carousel.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { NgbCarouselConfig } from '@ng-bootstrap/ng-bootstrap';
+@Component({
+  selector: 'app-carousel',
+  templateUrl: './carousel.component.html',
+  encapsulation: ViewEncapsulation.None,
+  styleUrls: ['./carousel.component.css'],
+  styles: [`
```

Comment:
why have not put this style in the `carousel.component.css` file??

Reply:
The slide component is a separate element and is not build with the rest of the html file. This means the styles must be developed in the component.